helpers/path.go

```diff
@@ -93,6 +93,18 @@ func MakeTitle(inpath string) string {
 	return strings.Replace(strings.TrimSpace(inpath), "-", " ", -1)
 }
 
+// Issue #2743
```

Comment:
Put the "Issue" reference at the end of the comment block to keep go-lint happy.

Also, can https://github.com/bep/inflect `Titleize` help solve this?